src/logic/widelands.cc

```diff
@@ -17,4 +17,26 @@
  *
  */
 
-// Dummy file as cmake cannot handle header only libraries :(.
+#include "logic/widelands.h"
+
+#include "io/streamread.h"
+#include "io/streamwrite.h"
+
+namespace Widelands {
+
+// All these values are used as weighting factors by Request code.
+const WarePriority WarePriority::kLow(1);
```

Comment:
`uint32_t` assignment is exception-safe, so if you made the constructor of `WarePriority` `noexcept`, we might be able to use `constexpr` here.

Reply:
https://stackoverflow.com/questions/29432283/c-static-constexpr-field-with-incomplete-type